### DISCUSSION TITLE
[ControlNet 1.1 Tile] If you have color offset problems with anime images

### DISCUSSION THREAD

**Discussion by lllyasviel** [Mikubill/sd-webui-controlnet#1163] [General]
Multiple users reported that the Tile 1.1 has color offset problems with anime images/models (it seems that realistic images/models does not have obvious problems).

You can solve the color offset problems by using one of the below methods

- Use higher cfg-scale (bigger than 15)
- Use your webui’s Danbooru Interrogator to generate prompts (or WD14 tagger if you wish). The generated prompts usually lead to reduced color offsets and much more accurate color preservation.
- Wait for us to release [control_v11f1e_sd15s2_tile_anime]. However, given our current workloads, the possibility of us releasing this model is less than 20%.
- Ignore the color offset problem (!). Just imagine that you are using midjourney (a system that process images in multiple stages and you do not know how many changes and “color offsets” are applied to “intermediate” images – you cannot even see those “intermediate” images.) If you can pretend that you have never seen the smaller image, you will forget the color offsets. Just pretend that you have never seen the smaller image, and appreciate the bigger image with great details, just like people appreciate midjourney’s black box. (But this may only work when the small image is not very important and can be ignored.)

Finally, another option is using lower denoising strength, but that is just “cheating” and does not really display the capability of Tile 1.1. Perhaps the best solution is the above “Danbooru Interrogator” + cfg 15

**Comment by zengshuisheng123** [comment]
big house

**Comment by Rkkss** [comment]
is there anyway to avoid strong depth of field when using high cfg?  lower cfg does help but then upscaling is really mediocre. Feel like abit of dilemma.

**Comment by lllyasviel** [comment]
fixed in 1.1.195 tiled_colorfix
